### PR TITLE
Google's first submit sharing what we've developed.

### PR DESCRIPTION
**Pull request by jgoodknight**
Google's first submit sharing what we've developed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Methods for Estimating the Union of Multiple Sets
```
Comment (jgoodknight) [inline README.md:1]: I see a few spelling errors in here, my apologies, will make sure to fix those before submission

Comment (jgoodknight) [reply README.md:1]: I think I got them all, but please let me know if you find any

**Comment by jgoodknight** [comment]
Please note we are investigating test failures in the repo which pass in our environment but appear to fail in some other environments

The affected files are
src/estimators/tests/bloom_filters_test.py
src/estimators/tests/liquid_legions_test.py

**Comment by jgoodknight** [comment]
> Please note we are investigating test failures in the repo which pass in our environment but appear to fail in some other environments
> 
> The affected files are
> src/estimators/tests/bloom_filters_test.py
> src/estimators/tests/liquid_legions_test.py

we have resolved this issue.  Thank you!

**Comment by zxhuang** [comment]
LGTM in general with some comments. OK to merge.

**Pull request merged**